src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/mate";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "Qwerty123";
```

Comment:
It's not secure to keep database credentials hardcoded in the source code. Consider using a more secure way to store them, like environment variables or configuration files.